internal/kibana/server.go

```diff
@@ -74,7 +74,7 @@ func (c *Client) CreateServiceToken(ctx context.Context) (ServiceToken, error) {
 	defer span.End()
 
 	reqBody := `{}`
-	statusCode, respBody, _ := c.post(ctx, fmt.Sprintf("%s/service-tokens", FleetAPI), []byte(reqBody))
+	statusCode, respBody, _ := c.post(ctx, fmt.Sprintf("%s/service_tokens", FleetAPI), []byte(reqBody))
```

Comment:
@adam-stokes I'm not seeing any usage for the `CreateServiceToken` method. Should we remove it?

Reply:
We should discuss with agent/fleet team on what tests they would like to see with the ServiceToken usage